lib/appwrite/services/users.rb

```diff
@@ -40,15 +36,71 @@ def list(search: nil, limit: nil, offset: nil, cursor: nil, cursor_direction: ni
             )
         end
 
+        
         # Create a new user.
         #
-        # @param [string] user_id User ID. Choose your own unique ID or pass the string &quot;unique()&quot; to auto generate it. Valid chars are a-z, A-Z, 0-9, period, hyphen, and underscore. Can&#039;t start with a special char. Max length is 36 chars.
-        # @param [string] email User email.
-        # @param [string] password User password. Must be at least 8 chars.
-        # @param [string] name User name. Max length: 128 chars.
+        # @param [String] user_id User ID. Choose your own unique ID or pass the string &quot;unique()&quot; to auto generate it. Valid chars are a-z, A-Z, 0-9, period, hyphen, and underscore. Can&#039;t start with a special char. Max length is 36 chars.
+        # @param [String] email User email.
+        # @param [String] phone Phone number. Format this number with a leading &#039;+&#039; and a country code, e.g., +16175551212.
```

Comment:
the comments are bork

Reply:
These will be fixed in the next stable version